### ISSUE TITLE
Input field aria-invalid issue

### ISSUE THREAD

**Issue by srancour** [department-of-veterans-affairs/va.gov-cms#17542]
## Describe the defect
In prod there exists an issue where, when an issue is marked with an error with `aria-invalid="true"`, when a user focuses on the field and then moves focus away, the field is changing to `aria-invalid="false`. This causes issues for screen reader users as they will think the field is now correct even though the error state still exists on the field and it can confuse the user because they won't know if it's the same field they were on before that had the error or if it's a new field entirely.

## To Reproduce
1. Go to ['Create Benefits Detail Page'](https://prod.cms.va.gov/node/add/page)
2. Fill in the 'Page title' field with something that goes beyond the character limit
3. Fill in the 'Meta description' and 'Revision log message' fields with something so they're not blank
4. Press Save
5. Code shows `aria-invalid="true"` for the 'Page title' field has been set
6. Focus 'Page title' field
7. Tab away or change focus from field
8. On fade, code now shows `aria-invalid="false"` for the 'Page title' field has been set

## AC / Expected behavior
The field should continue to have `aria-invalid="true"` set on all fields while there is still an active error on it.

## Screenshots
This has been captured in [my comment on ticket #17449](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/17449#issuecomment-1998484579).

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

**Comment by gracekretschmer-metrostar** [comment]
Need to do: bring in a drupal developer to help with discovery.
